### DISCUSSION TITLE
Ethereum's provider isConnected method returns true always

### DISCUSSION THREAD

**Discussion by nbran** [phantom/docs#241] [Help & Support]
### Summary

Even for a dApp that has never been associated with the wallet before.

### Steps to Reproduce

1. Go to any dApp
2. Open developer tools 
3. In the console, execute `window.phantom.ethereum.isConnected();`
4. The response is always `true`

### Phantom Version

2.4.8

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
@nbran Thank you for flagging! We are investigating.

**Comment by adamdelphantom** [comment]
@nbran After discussing this I learned that window.ethereum.isConnected refer's to the provider's connection to RPC nodes, not about the user account. You can find more info here: https://ethereum.stackexchange.com/questions/134183/window-ethereum-isconnected-always-returns-true

**Comment by nbran** [comment]
Thanks, @adamdelphantom. I think that doesn't quite explain the issue as the function will return true even if there is no connection to the RPC node (you can replicate easily by calling the function offline). 

Furthermore, if the function is indeed designed to work as you describe for Ethereum/Polygon, then that would be inconsistent with the equivalents for Solana and Bitcoin which seems to return `true` when the wallet is connected to a dApp (including during offline / no access to RPC nodes).
